{{cookiecutter.github_repository_name}}/docs/api/users.md

```diff
@@ -7,11 +7,17 @@ Supports registering, viewing, and updating user accounts.
 
 `POST` `users/`
```

Comment:
If you ask me, this should be 

      `POST` `/users/` 

with the leading slash, somehow seems weird hanging w.o. it... but I let u decide. (same change for other endpoints, including in authorization.md)